igniter/bootstrap_repos.py

```diff
@@ -289,6 +294,31 @@ def get_openpype_path(cls):
         """
         return os.getenv("OPENPYPE_PATH")
 
+    @classmethod
+    def get_local_openpype_path(cls, settings=None):
```

Comment:
You're caching the value on class so if you'll don't pass settings on first call, and then pass it on second, it won't use the settings because it's already cached.

I don't think the function implementation should be classmethod of `OpenPypeVersion` and should cache the value if passed argument should affect output but does not.